README.md

```diff
@@ -132,17 +132,11 @@ The **Server Connection** in message proxy should be configured with the two def
 
 <img src="src/main/resources/static/common/images/mq_proxy_mode_diagram.png" alt="mq proxy mode diagram">
 
-### Using virtual asset with JMS and RabbitMQ
-To use the queueing system with virtual asset, you can change **Inventory service request queue** to a customized destination queue name.
```

Comment:
Let separate the text description for queueing system and Kafka (at line 142) since it doesn't read smoothly. 
And change the start of this paragraph to be "To use JMS and RabbitMQ with virtual asset".